informer/integration/basic_test.go

```diff
@@ -12,24 +12,29 @@ import (
 // informer operations. The test verifies the informer is operating as expected
 // when processing basic sequences of creating and deleting runtime objects.
 func Test_Informer_Integration_Basic(t *testing.T) {
-	testSetup(t)
-	defer testTeardown(t)
-
-	const timeDelta = time.Millisecond * 100
-
-	objectIDOne := "al7qy"
-	objectIDTwo := "al8qy"
+	mustSetup()
+	defer mustTeardown()
 
+	crIDOne := "al7qy"
+	crIDTwo := "al8qy"
```

Comment:
The object is configmap, so I think cmID is more accurate prefix.

Reply:
Done.